client/css/app.styl

```diff
@@ -309,6 +309,24 @@ body.heaven
     color:#0000ff
   figure.icon
     float left
+#players li.dead figure.icon .avatar
+  position:absolute
+  /* Firefox 10+, Firefox on Android */
+  filter: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnPjxmaWx0ZXIgaWQ9J2dyYXlzY2FsZSc+PGZlQ29sb3JNYXRyaXggdHlwZT0nbWF0cml4JyB2YWx1ZXM9JzAuMzMzMyAwLjMzMzMgMC4zMzMzIDAgMCAwLjMzMzMgMC4zMzMzIDAuMzMzMyAwIDAgMC4zMzMzIDAuMzMzMyAwLjMzMzMgMCAwIDAgMCAwIDEgMCcvPjwvZmlsdGVyPjwvc3ZnPg==#grayscale");
+  /* IE 6-9 */
+  filter: gray
+  /*
+    Chrome 19+,
+    Safari 6+,
+    Safari 6+ iOS,
+    Opera 15+
+    */
+  -webkit-filter: grayscale(100%)
```

Comment:
Could you also add a non-prefixed version (`filter: grayscale(100%)` )?
I think the `-webkit-` prefix should always be a kind of supplement to the non-prefixed one.
Also, `filter: gray` for IE 6-9 is not needed since we are not targeting that version of IE.